plugin/completion.py

```diff
@@ -35,10 +37,15 @@
 SessionName = str
 CompletionResponse = Union[List[CompletionItem], CompletionList, None]
 ResolvedCompletions = Tuple[Union[CompletionResponse, Error], 'weakref.ref[Session]']
+CompletionsStore = TypedDict('CompletionsStore', {
```

Comment:
I don't like this name,
if you have something else to suggest, I am for changing it.

It is only used in this file, so I am not that bothered by it.

Reply:
It's fine I think :)